README.md

```diff
@@ -66,7 +342,7 @@ Next extract key:
 openssl rsa -in `exampleName.pem` -out `exampleKey.pem`
 ```
 
-When starting  h2 worker pass key and cerificate to workers tls options:
+When starting  h2 worker pass key and certificate to workers tls options:
```

Comment:
When starting an h2 worker, pass the key and certificate to workers' tls options:

Reply:
From the perspective of the readme's reader, what is an "h2 worker"?

Reply:
This section is from pre API times, I'll remove it.